inc/dg/functors.h

```diff
@@ -514,7 +539,84 @@ struct SinX
   private:
     double amp_,bamp_,kx_;
 };
-
+/**
+ * @brief Functor for a sin prof in y-direction
+ * \f[ f(x,y) =B+ A \sin(k_y y) \f]
+ */
+struct SinY
+{
+    /**
+     * @brief Construct with two coefficients
+     *
+     * @param amp amplitude
+     * @param bamp backgroundamp
+     * @param kx  kx
+     */
+    SinY( double amp, double bamp, double ky):amp_(amp), bamp_(bamp),ky_(ky){}
+    /**
+     * @brief Return profile
+     *
+     * @param x x - coordinate
+     * @param y y - coordinate
+     
+     * @return \f$ f(x,y)\f$
+     */
```

Comment:
Könntest du hier noch ein DG_DEVICE einbauen, damits auf der GPU auch compiliert (so wie in SinXSinY z.B.)